### DISCUSSION TITLE
How to use pytest handlecrashitem

### DISCUSSION THREAD

**Discussion by dln22** [pytest-dev/pytest-xdist#811] [General]
Hello,

Could someone, please advise how to use the pytest_handlecrashitem hook to reschedule and rerun a crashed test? I have a rather rarely occuring issue in my test suite, where some workers crash leading to one test out of two thousand failing. It would be nice to be able to rerun just this single failed case rather than the entire test suite. I have tried the example in the pytest_handlecrashitem docs, marking test as pending etc. But it does not rerun the test. I wonder if it is possible without any extra plugins? Thank you!

**Comment by nicoddemus** [comment]
Perhaps https://github.com/pytest-dev/pytest-xdist/pull/813 fixes your issue?
